src/pluginManager.ts

```diff
@@ -340,7 +340,7 @@ export class PluginManager {
     }
 
     const plugin = new Plugin(name, absolutePath, packageJson, scope);
-    this.plugins.set(name, plugin);
+    this.plugins.set(packageJson.name, plugin);
```

Comment:
may use the `identifier` variable above? Not really important just a very minor aesthetic thing (as we also use above `const alreadyInstalled = this.plugins.get(*identifier*);`

Reply:
Good point. Done.